libyul/AsmAnalysis.cpp

```diff
@@ -694,7 +694,7 @@ void AsmAnalyzer::expectValidType(YulName _type, SourceLocation const& _location
 		m_errorReporter.typeError(
 			5473_error,
 			_location,
-			fmt::format("\"{}\" is not a valid type (user defined types are not yet supported).", _type)
+			fmt::format("\"{}\" is not a valid type (user defined types are not yet supported).", _type.str())
```

Comment:
Huh, doesn't the fmt formatter for `YulString` work anymore?

Reply:
In my local tests it doesn't. Didn't we changed it to `YulName`?

Reply:
The fmtlib formatter specialization in ``libyul/YulString.h`` no longer worked without change with the updated fmtlib - but we should either fix that or remove it :-). (Removing it would be fine, since we'll move away from it hopefully eventually anyways, but we shouldn't leave the dead code around here)

Reply:
In favor of remove!